src/chi/components/buttons/buttons.scss

```diff
@@ -533,30 +542,40 @@ $sizes: (
           border: $border solid transparent;
           box-shadow: none;
           color: set-color($color);
-        }
-
-        &:hover,
-        &.-hover {
-          background-color: set-color(transparent);
-          border: $border solid set-color($color);
-        }
 
-        &:focus,
-        &.-focus {
-          background-color: set-color(transparent);
-          border: $border solid set-color($color);
-          box-shadow: 0 0 0 3px rgba(set-color($color), 0.4);
-        }
+          &:hover,
+          &.-hover {
+            background-color: set-color(transparent);
+            border: $border solid set-color($color);
+          }
 
-        &:active,
-        &.-active {
-          &,
           &:focus,
           &.-focus {
-            background-color: transparent;
+            background-color: set-color(transparent);
             border: $border solid set-color($color);
-            box-shadow: none;
-            color: set-color($color);
+          }
+
+          &:active,
+          &.-active {
+            &,
+            &:focus,
+            &.-focus {
+              background-color: transparent;
+              border: $border solid set-color($color);
+              box-shadow: none;
```

Comment:
Still seeing a box-shadow inset on flat button :active:focus state. Need to set that to none. Also, we lost the outer box-shadow on focus and active for flat buttons.